plugins/modules/ovirt_disk.py

```diff
@@ -355,8 +355,7 @@ def create_transfer_connection(module, transfer, context, connect_timeout=10, re
         connection.connect()
     except OSError as e:
         # Typically ConnectionRefusedError or socket.gaierror.
-        module.warn("Cannot connect to {}, trying {}: {}"
-                    .format(transfer.transfer_url, transfer.proxy_url, e))
+        module.warn("Cannot connect to %s, trying %s: %s" % (transfer.transfer_url, transfer.proxy_url, e))
```

Comment:
This is the same as using {}. Please check module.warn() interface - does it work like Pyhon logging, accepting
fmt, *args or it a single srting?

Then check how other code in this area generate messages and use the same style.

Reply:
Oh that was your question it only accepts one string no other args. 
And it is only message the warning styling is done automatically.